packages/assertion-monitor/chains.ts

```diff
@@ -0,0 +1,54 @@
+import { Chain } from 'viem'
+import {
+  mainnet,
+  arbitrum,
+  arbitrumNova,
+  base,
+  sepolia,
+  holesky,
+  arbitrumSepolia,
+  baseSepolia,
+} from 'viem/chains'
+
+export const DEFAULT_TIME_SPAN_SECONDS = 60 * 60 * 24 * 7 // 1 week
+
+export const supportedParentChains = [
+  mainnet,
+  arbitrum,
+  arbitrumNova,
+  base,
+  sepolia,
+  holesky,
+  arbitrumSepolia,
+  baseSepolia,
+]
+
+export const getChainFromId = (chainId: number): Chain => {
+  const chain = supportedParentChains.filter(chain => chain.id === chainId)
+  return chain[0] ?? null
+}
+
+export const getBlockTimeForChain = (chain: Chain): number => {
+  switch (chain) {
+    case mainnet:
+    case sepolia:
+    case holesky:
+      return 12
+
+    case base:
+    case baseSepolia:
+      return 2
+
+    case arbitrum:
+    case arbitrumNova:
+    case arbitrumSepolia:
+      return 4
+
+    default:
+      return 1
+  }
+}
+
+export const getDefaultBlockRange = (chain: Chain): bigint => {
+  return BigInt(DEFAULT_TIME_SPAN_SECONDS / getBlockTimeForChain(chain))
```

Comment:
I think it's now dividing by 4 instead of multiplying (for Arbitrum chains). It should multiply in the case of Arbitrum and divide in the case of Ethereum, or base.

Either that, or return 0.25 in `getBlockTimeForChain` instead of 4